pallets/rewards/src/tests.rs

```diff
@@ -33,13 +33,31 @@ fn reward_amount_limit() {
 #[test]
 fn issue_reward_handling() {
 	new_test_ext().execute_with(|| {
-		Rewards::issue_reward(123456, 55);
+		Rewards::issue_reward(1, 55);
 
 		let event = <frame_system::Pallet<Test>>::events()
 			.pop()
 			.expect("Expected at least one EventRecord to be found")
 			.event;
 
-		assert_eq!(event, crate::mock::Event::Rewards(crate::Event::RewardIssued(123456, 55)),);
+		assert_eq!(event, crate::mock::Event::Rewards(crate::Event::RewardIssued(1, 55)),);
+	});
+}
+
+#[test]
+fn rewards_were_issued_after_mining_blocks() {
+	log::debug!("******* CAN WE SEE THIS");
```

Comment:
@nathanwhit I can see debug logs coming from other pallets (or their tests) but not from `rewards/src/tests.rs`. What am I doing wrong? 

```bash
$ export RUST_LOG=debug

$ cargo test -- --test-threads 1
   Compiling creditcoin-node-runtime v2.0.0-beta.5 (/home/ubuntu/creditcoin-substrate/runtime)
   Compiling pallet-creditcoin v2.0.0-beta.5 (/home/ubuntu/creditcoin-substrate/pallets/creditcoin)
   Compiling pallet-rewards v2.0.0-beta.5 (/home/ubuntu/creditcoin-substrate/pallets/rewards)
   Compiling creditcoin-node-rpc v2.0.0-beta.5 (/home/ubuntu/creditcoin-substrate/node/rpc)
   Compiling creditcoin-node v2.0.0-beta.5 (/home/ubuntu/creditcoin-substrate/node)
    Finished test [unoptimized + debuginfo] target(s) in 27.11s
     Running unittests src/lib.rs (target/debug/deps/creditcoin_node-b7423fa73ea48302)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/main.rs (target/debug/deps/creditcoin_node-d23303f8f140c94e)

running 1 test
test cli::parse_tests::parse_rpc_pair_quoted ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/creditcoin_node_rpc-496683024e00f191)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/creditcoin_node_runtime-6073869da3cfe26c)

running 1 test
test __construct_runtime_integrity_test::runtime_integrity_tests ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/pallet_creditcoin-088d807c29b66741)

running 81 tests
test mock::__construct_runtime_integrity_test::runtime_integrity_tests ... ok
test ocw::tests::eth_address_bad_hex ... ok
test ocw::tests::eth_address_bad_len ... ok
test ocw::tests::eth_address_non_utf8 ... ok
test ocw::tests::eth_address_valid ... ok
test ocw::tests::ethless_transfer_amount_mismatch ... ok
test ocw::tests::ethless_transfer_contract_mismatch ... ok
test ocw::tests::ethless_transfer_from_mismatch ... ok
test ocw::tests::ethless_transfer_invalid_input_data ... ok
test ocw::tests::ethless_transfer_to_mismatch ... ok
test ocw::tests::ethless_transfer_tx_ahead_of_tip ... ok
test ocw::tests::ethless_transfer_tx_failed ... ok
test ocw::tests::ethless_transfer_tx_unconfirmed ... ok
test ocw::tests::ethless_transfer_valid ... ok
test tests::add_add_ask_order_rejects_zero_maturity - should panic ... ok
test tests::add_ask_order_basic ... ok
test tests::add_ask_order_pre_existing ... ok
test tests::add_ask_order_used_guid ... ok
test tests::add_authority_errors_for_non_root ... ok
test tests::add_authority_works_for_root ... ok
test tests::add_bid_ask_order_rejects_zero_maturity - should panic ... ok
test tests::add_bid_order_basic ... ok
test tests::add_bid_order_pre_existing ... ok
test tests::add_bid_order_used_guid ... ok
test tests::add_deal_order_basic ... ok
test tests::add_deal_order_existing ... ok
test tests::add_offer_basic ... ok
test tests::add_offer_existing ... ok
test tests::claim_legacy_wallet_works ... ok
test tests::close_deal_order_should_error_when_borrower_address_is_not_registered ... ok
test tests::close_deal_order_should_error_when_deal_isnt_locked ... ok
test tests::close_deal_order_should_error_when_deal_order_has_already_been_repaid ... ok
test tests::close_deal_order_should_error_when_deal_timestamp_is_in_the_future ... ok
test tests::close_deal_order_should_error_when_not_signed ... ok
test tests::close_deal_order_should_error_when_not_signed_by_borrower ... ok
test tests::close_deal_order_should_error_when_transfer_amount_is_not_enough ... ok
test tests::close_deal_order_should_error_when_transfer_block_is_greater_than_current_block ... ok
test tests::close_deal_order_should_error_when_transfer_has_already_been_processed ... ok
test tests::close_deal_order_should_error_when_transfer_order_id_doesnt_match_deal_order_id ... ok
test tests::close_deal_order_should_error_when_transfer_sighash_doesnt_match_borrower ... ok
test tests::close_deal_order_should_succeed ... ok
test tests::exempt_should_error_for_non_lender ... ok
test tests::exempt_should_error_when_deal_order_has_already_been_repaid ... ok
test tests::exempt_should_error_when_not_signed ... ok
test tests::exempt_should_error_when_transfer_has_been_processed ... ok
test tests::exempt_should_error_when_transfer_order_id_doesnt_match_deal_order_id ... ok
test tests::exempt_should_succeed ... ok
test tests::fund_deal_order_should_error_for_non_lender ... ok
test tests::fund_deal_order_should_error_when_address_not_registered ... ok
test tests::fund_deal_order_should_error_when_deal_has_expired ... Mar 28 12:53:20.058 DEBUG pallet_creditcoin::pallet: Cleaning up expired entries    
Mar 28 12:53:20.059 DEBUG pallet_creditcoin::pallet: Cleaning up expired entries    
Mar 28 12:53:20.059 DEBUG pallet_creditcoin::pallet: Cleaning up expired entries    
Mar 28 12:53:20.059 DEBUG pallet_creditcoin::pallet: Cleaning up expired entries    
ok
test tests::fund_deal_order_should_error_when_deal_is_funded ... ok
test tests::fund_deal_order_should_error_when_not_signed ... ok
test tests::fund_deal_order_should_error_when_timestamp_is_in_the_future ... ok
test tests::fund_deal_order_should_error_when_transfer_amount_doesnt_match ... ok
test tests::fund_deal_order_should_error_when_transfer_has_been_processed ... ok
test tests::fund_deal_order_should_error_when_transfer_order_id_doesnt_match_deal_order_id ... ok
test tests::fund_deal_order_should_error_when_transfer_sighash_doesnt_match_lender ... ok
test tests::fund_deal_order_works ... ok
test tests::lock_deal_order_locks_by_borrower ... ok
test tests::lock_deal_order_should_error_for_non_existent_deal_order ... ok
test tests::lock_deal_order_should_error_when_not_funded ... ok
test tests::lock_deal_order_should_error_when_not_signed ... ok
test tests::lock_deal_order_should_fail_for_non_borrower ... ok
test tests::lock_deal_order_should_fail_if_already_locked ... ok
test tests::register_address_basic ... ok
test tests::register_address_pre_existing ... ok
test tests::register_deal_order_accepts_ed25519 ... ok
test tests::register_deal_order_accepts_sr25519 ... ok
test tests::register_deal_order_should_error_when_ask_order_id_exists ... ok
test tests::register_deal_order_should_error_when_bid_order_id_exists ... ok
test tests::register_deal_order_should_error_when_borrower_address_doesnt_match_signature ... ok
test tests::register_deal_order_should_error_when_deal_order_id_exists ... ok
test tests::register_deal_order_should_error_when_lender_address_doesnt_match_sender ... ok
test tests::register_deal_order_should_error_when_lender_and_borrower_are_on_different_chains ... ok
test tests::register_deal_order_should_error_when_not_signed ... ok
test tests::register_deal_order_should_error_when_offer_id_exists ... ok
test tests::register_deal_order_should_error_when_signature_is_invalid ... ok
test tests::register_deal_order_should_succeed ... ok
test tests::register_transfer_ocw ... Mar 28 12:53:20.510 DEBUG pallet_creditcoin::pallet: verifying transfer    
Mar 28 12:53:20.511 DEBUG pallet_creditcoin::pallet: verify_transfer result: Ok(())    
Mar 28 12:53:20.644 DEBUG runtime::offchain: Creating signed transaction from account: f63436d59a774f122cd9a8ebac59a2687c0c8bc4ce03bdfd78a1b7f61830643e (5HdX9Wxj...) (nonce: 0)    
Mar 28 12:53:20.645 DEBUG pallet_creditcoin::pallet: Cleaning up expired entries    
ok
test tests::verify_ethless_transfer ... ok
test types::loan_terms::tests::test_calc_interest ... ok

test result: ok. 81 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.85s

     Running unittests src/lib.rs (target/debug/deps/pallet_difficulty-922cc5bdccd000d8)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/pallet_rewards-a7b769daaf8601b1)

running 9 tests
test mock::__construct_runtime_integrity_test::runtime_integrity_tests ... ok
test tests::issue_reward_handling ... ok
test tests::reward_amount_after_one_halflife ... ok
test tests::reward_amount_after_two_halflives ... ok
test tests::reward_amount_block_one ... ok
test tests::reward_amount_genesis ... ok
test tests::reward_amount_just_before_reaching_first_halflife ... ok
test tests::reward_amount_limit ... ok
test tests::rewards_were_issued_after_mining_blocks ... ok

test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.01s

     Running unittests src/lib.rs (target/debug/deps/primitives-e58b41aa312254a8)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/sha3pow-14c3f0a32052bc1a)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests creditcoin-node

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests creditcoin-node-rpc

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests creditcoin-node-runtime

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests pallet-creditcoin

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests pallet-difficulty

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests pallet-rewards

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests primitives

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests sha3pow

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

Reply:
Pushed a commit to enable the logging. Side note: if you want to run only the tests for a single crate (e.g. pallet-rewards) you can pass it as the `--package` like `cargo test --package pallet-rewards`.